gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/SharedPolicyGroupHistoryRepository.java

```diff
@@ -21,10 +21,13 @@
 import io.gravitee.repository.management.api.search.SharedPolicyGroupHistoryCriteria;
 import io.gravitee.repository.management.api.search.Sortable;
 import io.gravitee.repository.management.model.SharedPolicyGroup;
+import java.util.Optional;
 
 public interface SharedPolicyGroupHistoryRepository extends CrudRepository<SharedPolicyGroup, String> {
     Page<SharedPolicyGroup> search(SharedPolicyGroupHistoryCriteria criteria, Pageable pageable, Sortable sortable)
         throws TechnicalException;
 
-    Page<SharedPolicyGroup> searchLatestBySharedPolicyPolicyGroupId(String environmentId, Pageable pageable) throws TechnicalException;
+    Page<SharedPolicyGroup> searchLatestBySharedPolicyGroupId(String environmentId, Pageable pageable) throws TechnicalException;
+
+    Optional<SharedPolicyGroup> getLatestBySharedPolicyGroupId(String environmentId, String sharedPolicyGroupId) throws TechnicalException;
```

Comment:
Maybe: `getLatestVersionBySharedPolicyGroupId`?

Reply:
humm, but that means I'm using the version property to take the "last" one, whereas we're using the updatedAt. (that seems safer to me) 🤔

Reply:
So I'd suggest `getLastUpdatedSharedPolicyGroupById` - this would be more intuitive. wdym?

Reply:
@ThibaudAV @dev-marek 
indeed here it is just about retrieving the latest entry of history for a given SPG id.
for me `Latest` is the good wording here